client/app/configuration/configuration.html

```diff
@@ -28,7 +49,7 @@ <h2 class="oui-tile__title" data-ng-bind="'core_type_of_guide_' + key | translat
                             </li>
                             <li class="oui-tile__item">
                                 <div class="oui-tile__definition">
-                                    <a data-ng-href="{{ ConfigurationCtrl.allGuides }}"
+                                    <a data-ng-href="{{ $ctrl.allGuides }}"
```

Comment:
```suggestion
                                     <a data-ng-href="{{:: $ctrl.allGuides }}"
```

Reply:
Nop, loaded asynchronously and when adding `::`, it didn't work ! A bit frustrated by the way `::` works sometimes, as documentation says that `undefined` variables will be tracked until they are